streams/src/test/java/org/apache/kafka/streams/integration/RegexSourceIntegrationTest.java

```diff
@@ -141,6 +138,8 @@ public void tearDown() throws IOException {
 
     @Test
     public void testRegexMatchesTopicsAWhenCreated() throws Exception {
+        final String outputTopic = createTopic("_1");
```

Comment:
Instead of manual counting, should we still do this in `setup()` and add `private static volatile AtomicInteger testNo = new AtomicInteger(0);` (cf. `GlobalKTableEOSIntegrationTest`)